src/test/java/com/google/api/codegen/testdata/python_doc_main_library.baseline

```diff
@@ -695,6 +709,7 @@ class LibraryServiceClient(object):
           :exc:`google.gax.errors.GaxError` if the RPC is aborted.
           :exc:`ValueError` if the parameters are invalid.
         """
+        # Create the request object.
```

Comment:
These comments fall in the category of unnecessary. They indicate what the code is doing rather than why. See recent TOTT.